### PR TITLE
feat: add hide-items-per-page props in v-data-footer

### LINKED ISSUE #9531: [Feature Request] show-items-per-page
### Problem to solve
user can't hide items-per-page option in v-data-footer

### Proposed solution
add new props 
name:  show-items-per-page,
type: boolean,
default: true

### PR DESCRIPTION
**Pull request by mahammad8564**
fixes #9531

## How Has This Been Tested?

visually
## Markup:

<details>

```vue
<template>
  <v-container>
    <v-data-table
      :headers="headers"
      :items="desserts"
      :items-per-page="5"
      class="elevation-1"
      :footer-props="{ hideItemsPerPage:true }"
    ></v-data-table>
  </v-container>
</template>

<script>
export default {
  data: () => ({
    headers: [
      {
        text: "Dessert (100g serving)",
        align: "left",
        sortable: false,
        value: "name"
      },
      { text: "Calories", value: "calories" },
      { text: "Fat (g)", value: "fat" },
      { text: "Carbs (g)", value: "carbs" },
      { text: "Protein (g)", value: "protein" },
      { text: "Iron (%)", value: "iron" }
    ],
    desserts: [
      {
        name: "Frozen Yogurt",
        calories: 159,
        fat: 6.0,
        carbs: 24,
        protein: 4.0,
        iron: "1%"
      },
      {
        name: "Ice cream sandwich",
        calories: 237,
        fat: 9.0,
        carbs: 37,
        protein: 4.3,
        iron: "1%"
      },
      {
        name: "Eclair",
        calories: 262,
        fat: 16.0,
        carbs: 23,
        protein: 6.0,
        iron: "7%"
      },
      {
        name: "Cupcake",
        calories: 305,
        fat: 3.7,
        carbs: 67,
        protein: 4.3,
        iron: "8%"
      },
      {
        name: "Gingerbread",
        calories: 356,
        fat: 16.0,
        carbs: 49,
        protein: 3.9,
        iron: "16%"
      },
      {
        name: "Jelly bean",
        calories: 375,
        fat: 0.0,
        carbs: 94,
        protein: 0.0,
        iron: "0%"
      },
      {
        name: "Lollipop",
        calories: 392,
        fat: 0.2,
        carbs: 98,
        protein: 0,
        iron: "2%"
      },
      {
        name: "Honeycomb",
        calories: 408,
        fat: 3.2,
        carbs: 87,
        protein: 6.5,
        iron: "45%"
      },
      {
        name: "Donut",
        calories: 452,
        fat: 25.0,
        carbs: 51,
        protein: 4.9,
        iron: "22%"
      },
      {
        name: "KitKat",
        calories: 518,
        fat: 26.0,
        carbs: 65,
        protein: 7,
        iron: "6%"
      }
    ]
  })
};
</script>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by mahammad8564** [comment]
No need to add new props hide-items-per-page,

we can hide items-per-page by just passing a blank array in items-per-page-options

`:footer-props="{ itemsPerPageOptions:[] }"`

File: packages/docs/src/lang/eo-UY/components/DataIterators.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     },
     "v-data-footer": {
       "disableItemsPerPage": "crwdns41102:0crwdne41102:0",
+      "hideItemsPerPage": "translate('Hide items-per-page dropdown')",
```
Comment (mahammad8564) [inline packages/docs/src/lang/eo-UY/components/DataIterators.json]: I don't know how to translate in eo-UY that's why

Comment (MajesticPotatoe) [reply packages/docs/src/lang/eo-UY/components/DataIterators.json]: you dont. you only need to add to `en`. `eo-UY` gets generated from crowdin when we upload `en` for translating

Comment (mahammad-sixberries) [reply packages/docs/src/lang/eo-UY/components/DataIterators.json]: ok, thanks for the update

**Pull request closed without merging**